WordPress/AbstractFunctionRestrictionsSniff.php

```diff
@@ -150,24 +150,32 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
 
 			if ( false !== $prev ) {
 				// Skip sniffing if calling a same-named method, or on function definitions.
-				if ( in_array( $tokens[ $prev ]['code'], array( T_FUNCTION, T_DOUBLE_COLON, T_OBJECT_OPERATOR ), true ) ) {
+				$skipped = array(
+					T_FUNCTION => true,
```

Comment:
I think you could do `T_FUNCTION => T_FUNCTION`. It would be nice if the values were at the same height,